phpunit/class-prepare-for-js-test.php

```diff
@@ -10,26 +10,15 @@
  */
 class Prepare_For_JS_Test extends WP_UnitTestCase {
 
-	function setUp() {
-		parent::setUp();
-
-		$this->reset();
-	}
-
 	function tearDown() {
 		parent::tearDown();
 
-		$this->reset();
-	}
-
-	function reset() {
-		foreach ( WP_Block_Type_Registry::get_instance()->get_all_registered() as $name => $block_type ) {
-			WP_Block_Type_Registry::get_instance()->unregister( $name );
-		}
+		$registry = WP_Block_Type_Registry::get_instance();
+		$registry->unregister( 'core/dummy' );
```

Comment:
If we assume all tests should register a block only with this name, it might make for a slight improvement to move it to a static variable of the class, e.g. `protected static $block_name = 'core/dummy';`